cardano-node/src/Cardano/Node/Run.hs

```diff
@@ -220,7 +220,8 @@ handleSimpleNode p trace nodeTracers npm onKernel = do
       let chainDB = getChainDB nodeKernel
       void $ onEachChange registry identity Nothing
                           (ChainDB.getTipPoint chainDB) $ \tip ->
-        atomically $ writeTVar varTip tip
+        -- TODO: use StrictTVar
+        atomically $ writeTVar varTip $! tip
```

Comment:
This patch is no longer necessary. The node top level no longer reads the tip at all for use in tracers.

Reply:
I used it today to trace epoch boundary into `eventlog`, but this could be done in `consensus` as well (it was just simpler to do that here, in bare `IO` :) with out any abstract constraints)